docs/src/components/home/MuiStatistics.tsx

```diff
@@ -4,10 +4,10 @@ import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography';
 
 const data = [
-  { title: '4.0M', metadata: 'Weekly downloads on npm' },
-  { title: '83k', metadata: 'Stars on GitHub' },
+  { title: '4M', metadata: 'Weekly downloads on npm' },
+  { title: '82.9k', metadata: 'Stars on GitHub' },
```

Comment:
It's 83003 right now:

```suggestion
  { title: '83k', metadata: 'Stars on GitHub' },
```